### PR TITLE
Logoperators

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: logoper.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        }
+    </script>
+
+    <script>
+        let login = prompt("Кто там?");
+        if (login === "Админ") {
+            let password = prompt("Пароль?");
+            if (password === "Я главный") {
+                alert("Здравствуйте")
+            } else if (password === " " || password === null) {
```
Comment (kathrynowy) [inline logoper.html:32]: remove this  === ' '
if i write 3 spaces?

rewrite condition without ' ' and null etc.

File: logoper.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                alert("Отменено");
+            } else {
+                alert("Неверный пароль");
+            }
+        } else if (login === " " || login === null) {
```
Comment (kathrynowy) [inline logoper.html:37]: same here

File: logoper.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        if (a > 0 && a < 5) {
+            alert("True");
+        } else {
+            alert("False");
+        }
+    </script>
+
+    <script>
+        let a = prompt(" Введите значение переменной а");
+        if (a === 0 || a === 2) {
```
Comment (kathrynowy) [inline logoper.html:55]: rewrite condition , remove === 0 , use another way

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: logoper.html (hunk 3 above)
Comment (Kartynnikk) [reply logoper.html:55]: fixed

File: logoper.html (hunk 1 above)
Comment (Kartynnikk) [reply logoper.html:32]: fixed

File: logoper.html (hunk 2 above)
Comment (Kartynnikk) [reply logoper.html:37]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**